src/smc_mallows_new_users_complete.cpp

```diff
@@ -146,6 +149,10 @@ Rcpp::List smc_mallows_new_users_complete(
     const double& maxw = arma::max(log_inc_wgt);
     const arma::vec& w = arma::exp(log_inc_wgt - maxw);
     const arma::vec norm_wgt = w / arma::sum(w);
+    
+    /* store ESS */
+    ESS_vec(tt) = 1/sum(norm_wgt^2);
```

Comment:
This may also generate a class conflict: `norm_wgt` is a double, while 1 and 2 are integers. The following might work better:

```
ESS_vec(tt) = 1.0 / sum(norm_wgt ^ 2.0);
```

Reply:
Thank you both for the helpful advice and guidance - I shall implement these changes now.

Apologies for my slow reply until now - I made these push requests just before I went on holiday for Easter. However, I could not find a secure enough WiFi connection to work on these changes. I'm also not keen on having GH or work emails on my phone...